src/react/components/pages/projectSettings/projectSettingsPage.scss

```diff
@@ -4,11 +4,17 @@
     height: 100%;
 
     &-settings {
-        flex-grow: 1;
+        flex-grow: 2;
     }
 
     &-metrics {
-        align-content: center;
-        align-items: center;
+        flex: 2 1 400px;
```

Comment:
Let's not use physical pixels - instead let's use `vw` units.  See app settings, connection, recent projects SCSS for example.

Reply:
Also consider specifying a min/max width.